packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -166,6 +167,8 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
 
   const messageThreadProps = usePropsFor(MessageThread);
   const sendBoxProps = usePropsFor(SendBox);
+  /* @conditional-compile-remove(rich-text-editor) */
+  const richTextSendBoxProps = usePropsFor(SendBox);
```

Comment:
We probably need to use `usePropsFor(RichTextSendBox)` here instead

Reply:
Oh right, meant to add that! Will update the PR